docs/framework_plugins.rst

```diff
@@ -1,5 +1,5 @@
-Framework integration
-=====================
+DL Framework Plugins
```

Comment:
I wonder if we want to change this that way. It is about how to use DALI in a given FW, so I wonder if we want to name it as a plugin doc.

Reply:
It's in the Python API documentation, and it contains the documentation of the plugins (for example, "MXNet Plugin API reference")

Reply:
OK. Lets us leave it as it is now.